### PR TITLE
Install bioblend with fixes

### PR DESCRIPTION
**Pull request by mvdbeek**
This bioblend fork actually installs the test data and sets the correct exit code if tests are failing. (https://github.com/galaxyproject/bioblend/pull/238)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,7 +47,7 @@ before_install:
   galaxy_kickstart`
 - docker ps
 install:
-- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend[testing]'
+  - sudo su $GALAXY_TRAVIS_USER -c 'pip install --user https://github.com/mvdbeek/bioblend/archive/test_data.zip pytest'
```
Comment (drosofff) [inline .travis.yml]: what about
`- sudo su $GALAXY_TRAVIS_USER -c 'cd && git clone https://github.com/mvdbeek/bioblend.git --branch test_data && pip install bioblend/ --user ` 
and we revert to
`- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend` when your PR is merged ?

Comment (mvdbeek) [inline .travis.yml]: Fine by me (except we'll have to manually install pytest), but why ?

Comment (drosofff) [inline .travis.yml]: Ah ! Indeed because I though this way avoids to put pytest in the pip install command. But I may be mixed...

Comment (mvdbeek) [inline .travis.yml]: OK, I see. This `pip install --user bioblend[testing]` will cause the extra dependencies of bioblend to be installed (see https://github.com/galaxyproject/bioblend/blob/master/setup.py#L40). But this syntax doesn't work when you point pip to a url, hence the explicit installation of pytest.

Comment (drosofff) [inline .travis.yml]: OK I get it ! thanks

**Pull request merged**